simplq/src/styles/footer.module.scss

```diff
@@ -1,128 +1,127 @@
 @import 'utils';
 
 .footer {
```

Comment:
I didn't make any modifications here, so it should just be eslint formatting.

Reply:
Yeah, I wanted to see if we can fix it. We are getting a lot of new first time collaborators due to hactober fest, and asking everyone to fix formatting before merging might be pain... so we will have to periodically fix the formatting. But fixing the formatting as an independent PR would make code reviews easier.